### PR TITLE
feat: [pipe-24078]: fix highlighting

### PR DESCRIPTION
**Pull request by cjlee01**
fix highlighting in diff viewer
image

### CODE REVIEW

File: apps/gitness/src/styles.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
 @import '@harnessio/ui/styles.css';
 /* @import '@harnessio/views/src/styles.css';
 @import '@harnessio/canary/src/styles.css'; */
+
+@import 'highlight.js/styles/atom-one-dark.css';
```
Comment (vardanbansal-harness) [inline apps/gitness/src/styles.css:5]: Will this css be also intelligently optimized by tailwind? Just wanted to know. This css might have a lot of styles but only few might actually be used in code. We should verify what all goes from it in the final style css for the app.

Comment (cjlee01) [reply apps/gitness/src/styles.css:5]: totally true, might be more than one style. I'm not sure if it'll be optimized by tailwind

Comment (knagurski) [reply apps/gitness/src/styles.css:5]: We almost certainly don't want Tailwind to touch this file at all

Comment (cjlee01) [reply apps/gitness/src/styles.css:5]: what do u recommend? @knagurski

File: apps/gitness/package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "@hookform/resolvers": "^3.6.0",
     "@tanstack/react-query": "4.36.1",
     "clipboard-copy": "^3.1.0",
+    "highlight.js": "^11.9.0",
```
Comment (abhinavrastogi-harness) [inline apps/gitness/package.json:32]: can you please link the sec approval ticket here?

File: packages/ui/src/views/repo/pull-request/diff-viewer/pull-request-diff-viewer.tsx [hunk 3]
```diff
@@ -64,6 +65,8 @@ const PullRequestDiffViewer = ({
 }: PullRequestDiffviewerProps) => {
   const ref = useRef<{ getDiffFileInstance: () => DiffFile }>(null)
   const valRef = useRef('')
+  const [_loading, setLoading] = useState(false)
```
Comment (abhinavrastogi-harness) [inline packages/ui/src/views/repo/pull-request/diff-viewer/pull-request-diff-viewer.tsx:68]:
```suggestion
  const [, setLoading] = useState(false)
```

File: apps/gitness/package.json (hunk 2 above)
Comment (cjlee01) [reply apps/gitness/package.json:32]: https://harness.atlassian.net/browse/SECAPP-1451

**Comment by cjlee01** [comment]
test

**Review by vardanbansal-harness — APPROVED** [review]

**Comment by cjlee01** [comment]
tet

**Pull request merged**